docs/resources/project.md

```diff
@@ -93,6 +93,7 @@ resource "vercel_project" "example" {
 
 Optional:
 
+- **git_branch** (String) The git branch of the environment variable.
```

Comment:
I'm not 100% familiar with this API, but can you only specify `git_branch` for `preview` `target`s, or does it work across all different target possibilities?
e.g. what happens if I specified
```
target     = ["production", "preview"]
git_branch = "staging" 
key        = "foo"
value      = "bar" 
```

I'm wondering if it's worth adding some additional validation around that. 
Granted the provider is currently pretty light on up-front validation.

Reply:
The API is very strict; the branch has to actually exist, can't be your production branch etc. Lots of things we cannot validate here. You get good error messages if you try to do something impossible.